src/exportToHtml.ts

```diff
@@ -75,12 +75,12 @@ export const exportToHtml = async (scenes: Scene[]) => {
   });
   html += `</div>
             <div id="navigation">
-              <button class="navbutton" type="button" onClick="moveLeft()">&#9664;</button>
+              <button class="navbutton" type="button" onClick="moveLeft()" title="Previous slide">&#9664;</button>
               <div id="title">1 of ${scenes.length}</div>
-              <button class="navbutton" type="button" onClick="moveRight()">&#9654;</button>          
+              <button class="navbutton" type="button" onClick="moveRight()" title="Next slide">&#9654;</button>          
               <div id="rightbuttons">
-                <button type="button" onClick="toggleMaximise()">&#x26F6</button>
-                <button type="button" onClick="closeNavigation()">&#x2716;</button>
+                <button type="button" onClick="toggleMaximise()" title="Toggle full-screen">&#x26F6</button>
```

Comment:
```suggestion
                <button type="button" onClick="toggleMaximise()" title="Toggle full-screen">&#x26F6;</button>
```

Actually, my browser (Chrome) didn't show the correct icon before. Does this fix it?

Reply:
Not sure, it works without the ; in my Chrome browser.  Can you check the new version?

Reply:
sorry, forgot to use the "Commit suggestion" button